schema/url/plots.raml

```diff
@@ -307,9 +299,7 @@ types:
     properties:
       completeCasesAllVars: number
       completeCasesAxesVars: number
-      computedVariableMetadata:
-        type: ComputedVariableMetadata
-        required: false
+      variables: VariableMetadata[]
```

Comment:
How would I know to look for the raml definition of `VariableMetadata` in EdaCommon?